### PR TITLE
I'd like to interject for a moment...

### PR DESCRIPTION
**Pull request by dvdmuckle**
Let me know if this is up to snuff.

### CODE REVIEW

**Comment by dvdmuckle** [comment]
This isn't up to snuff. Will make up to snuff.

File: lib/module/interject.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/dvdmuckle/irkbot/lib/configure"
+	"github.com/dvdmuckle/irkbot/lib/message"
+	"strings"
+)
+
+func HelpInterject() []string {
+	// This function returns an array of strings describing this command's
+	// functionality. It is displayed when someone gives the help command in a
+	// channel or private message.
+	s := "interject <subject> - return a popular copypasta defaulting to our favorite operating system"
```
Comment (davidscholberg) [inline lib/module/interject.go]: It should be `[subject]`, to indicate that the subject is optional.

Also, I'd change the description to: `give a pedantic rant about the proper way to refer to the given subject, defaulting to Linux if no subject given`. But that string is probably too long for one line, so please break it up using the `+` operator.

File: lib/module/interject.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	return []string{s}
+}
+
+func Interject(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	var interMsg string
+	if len(in.MsgArgs[1:]) > 0 {
+		interMsg = strings.Join(in.MsgArgs[1:], " ")
+	} else {
+		interMsg = "Linux"
+	}
```
Comment (davidscholberg) [inline lib/module/interject.go]: you can simplify this a little by initializing `interMsg` to `Linux`, e.g.:

```go
interMsg := "Linux"
if len(in.MsgArgs[1:]) > 0 {
	interMsg = strings.Join(in.MsgArgs[1:], " ")
}
```

File: lib/module/interject.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	msg := fmt.Sprintf("I'd just like to interject for a moment. What you're refering to as %s, is in fact, GNU/%s, or as I've recently taken to calling it, GNU plus %s.", interMsg, interMsg, interMsg)
```
Comment (davidscholberg) [inline lib/module/interject.go]: This line is a bit too long. I generally try to keep the lines under 80 characters where practical, and it's certainly practical to do that with a long string if you use the `+` operator (of which there are example usages elsewhere in irkbot).

**Review by davidscholberg — CHANGES_REQUESTED** [review]
Just a few small changes.

File: lib/module/interject.go (hunk 3 above)
Comment (davidscholberg) [reply lib/module/interject.go]: Oh, also, I think that the `GNU/` should be changed to `GNU *slash*`, and `GNU plus` should be changed to `GNU *plus*`. I think that better fits the intended emphases.

**Comment by dvdmuckle** [comment]
Shortened strings and cleaned up interMsg init, let me know if that's good.

**Comment by davidscholberg** [comment]
Changes look good. Please fix the imports and squash your commits down to one.

**Review by davidscholberg — APPROVED** [review]

**Pull request merged**